pkgs/development/libraries/openssl/default.nix

```diff
@@ -87,6 +87,7 @@ let
         x86_64-linux = "./Configure linux-x86_64";
         x86_64-solaris = "./Configure solaris64-x86_64-gcc";
         powerpc64-linux = "./Configure linux-ppc64";
+        riscv32-linux = "./Configure linux-latomic";
```

Comment:
Do you have a source for linux-latomic by chance? 😅

Reply:
WDYM? I figured this out myself — linux-generic32 causes build failures because of missing atomic symbols, and linux-latomic is linux-generic32 except it links libatomic, which provides those symbols.  It's defined here:

https://github.com/openssl/openssl/blob/16e7da091c67e072b6927fcbf8637824bfd13f83/Configurations/10-main.conf#L698-L701

Reply:
Ah, the riscv32 epxands on that. Shouldn't we use that?https://github.com/openssl/openssl/blob/16e7da091c67e072b6927fcbf8637824bfd13f83/Configurations/10-main.conf#L825

Reply:
Oh, yes we should! I didn't see that.

Reply:
Ah, I didn't see it because our default OpenSSL is too old for that.